scopes/react/component-highlighter/component-highlighter/hover-highlighter/hover-highlighter.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import classnames from 'classnames';
+import { useDebouncedCallback } from 'use-debounce';
+import { domToReact, toRootElement } from '@teambit/react.modules.dom-to-react';
+import { HoverSelector } from '@teambit/react.ui.hover-selector';
+import { hasComponentMeta } from '../bit-react-component';
+
+import styles from './hover-highlighter.module.scss';
+import { excludeHighlighterSelector } from '../../ignore-highlighter';
+import { ElementHighlighter, HighlightTarget, Placement, HighlightClasses } from '../../element-highlighter';
+
+export interface HoverHighlighterProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
the component is a fragment so I dont think we should the base type of div

Reply:
I'm forwarding props to `HoverSelector`, and I don't want to use `HoverSelectorProps`